admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue

```diff
@@ -247,6 +247,13 @@
             </div>
         </div>
     </div>
+
+      <admin-components-publishingmodal
+          v-if="publishDialogPath"
+          :isOpen="publishDialogPath"
+          :path="publishDialogPath"
+          @complete="closePublishing"
+          :modalTitle="`Web Publishing: ${publishDialogPath.split('/').pop()}`" />
```

Comment:
What happens if `publishDialogPath` is `null/undefined`?
Is that even possible?

Reply:
Nothing thanks to `v-if` above. And yes - we explicitly set it to `null` in `closePublishing` below.

Reply:
man im blind..